PY-SDK/admin/instructions/init_market.py

```diff
@@ -0,0 +1,105 @@
+from unicodedata import category
+from ...public.src.pyaver.market import AverMarket
```

Comment:
Are we sure we want to import here from ... instead of pyaver? What do we think?